types/level/index.d.ts

```diff
@@ -0,0 +1,18 @@
+// Type definitions for level 6.0
+// Project: https://github.com/Level/level
+// Definitions by: danielfx90 <https://github.com/danielfx90>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import { AbstractOptions, ErrorCallback } from 'abstract-leveldown';
+
+import EncodingDown from 'encoding-down';
+
+import { LevelUp } from 'levelup';
+
+export interface LevelDB<K = any, V = any> extends LevelUp<EncodingDown<K, V>> {
+  errors: /* typeof levelerrors */ any; // ? level-errors is not in DT
+}
+
+declare function level(location: string, options?: AbstractOptions, callback?: ErrorCallback): LevelDB;
+
+export default level;
```

Comment:
IMO this look like proper CommonJS package (4 years old export):
https://github.com/Level/level/blob/master/level.js#L1.
This is not ESM (though it could be used as such when consuming types)
```suggestion
export = level;
```

Reply:
@peterblazejewicz I committed your suggestions but now I realize that I also have the `LevelDB` interface exported. With `export = level;` I cannot have anything else exported.

How do you recommend solving this issue?

Reply:
Nevermind, I found out how. Sorry